### ISSUE TITLE
Compiling for Mac gives error [-Werror,-Wunused-but-set-variable]

### ISSUE THREAD

**Issue by aptonline** [VitaSmith/cdecrypt#8]
When running `make` in the cdecrypt folder I get the following errors:

```
[C] cdecrypt.c
cdecrypt.c:220:14: error: variable 'written' set but not used [-Werror,-Wunused-but-set-variable]
    uint64_t written = 0;
             ^
cdecrypt.c:304:14: error: variable 'written' set but not used [-Werror,-Wunused-but-set-variable]
    uint64_t written = 0;
             ^
2 errors generated.
make: *** [cdecrypt.o] Error 1
```

**Comment by aandnota** [comment]
Got these same errors today.

`cdecrypt.c:220:14: error: variable 'written' set but not used [-Werror,-Wunused-but-set-variable]
    uint64_t written = 0;
             ^
cdecrypt.c:304:14: error: variable 'written' set but not used [-Werror,-Wunused-but-set-variable]
    uint64_t written = 0;
             ^
2 errors generated.
make: *** [cdecrypt.o] Error 1`

**Comment by VitaSmith** [comment]
Please clone the latest git source. This has been fixed in git (#7).

**Comment by aptonline** [comment]
Worked a treat, thanks for the heads up :)
